data/json/items/armor/integrated.json

```diff
@@ -213,7 +222,16 @@
     "color": "black",
     "warmth": 45,
     "environmental_protection": 1,
-    "flags": [ "INTEGRATED", "ALLOWS_NATURAL_ATTACKS", "UNBREAKABLE", "SKINTIGHT", "NORMAL", "HELMET_COMPAT", "WATER_FRIENDLY" ],
+    "flags": [
+      "INTEGRATED",
+      "ALLOWS_NATURAL_ATTACKS",
+      "UNBREAKABLE",
+      "SKINTIGHT",
+      "NORMAL",
+      "HELMET_COMPAT",
+      "WATER_FRIENDLY",
+      "SOFT"
```

Comment:
Was this rigid before? It's made of just fur so it should already be soft I think!

Reply:
![image](https://user-images.githubusercontent.com/94415528/166332661-49450c2e-10c4-4983-93f5-fa8f7fc75099.png)
![image](https://user-images.githubusercontent.com/94415528/166332713-6ee7a027-00f1-4ee0-aae3-fb3f0a1ffc0e.png)

It doesn't say anything about rigidity, but it prevents wearing rigid items. Does helmet_compat imply rigidity?

Reply:
thanks for showing me. This is definitely  a bigger issue bug with footwear then, I'll make note of it personally and look into it when I get a chance. I think your "SOFT" solution is perfectly fine for now, and I'll look into why this is happening in the future.